inc/Smartling/Jobs/JobAbstract.php

```diff
@@ -162,7 +164,11 @@ public function runCronJob(string $source = ''): void
             $this->getLogger()->debug($message);
         } catch (SmartlingApiException $e) {
             $errorMessage = $e->getErrors()[0]['message'] ?? $e->getMessage();
-            $message = "Failed to place lock flag for {$this->getJobHookName()}: $errorMessage";
+            if (count($e->getErrorsByKey('resource.locked')) > 0) {
+                $message = "Unable to start cron, cron job already working, try again later";
```

Comment:
Avoid guessing what job can't start.
```suggestion
                $message = "Unable to start {$this->getJobHookName()} cron, cron job already working, try again later";
```